packages/db/package.json

```diff
@@ -39,7 +39,7 @@
     "is-valid-hostname": "1.0.2",
     "knex": "2.4.1",
     "lodash": "^4.17.21",
-    "pg": "^8.3.0",
+    "pg": "~8.10.0",
```

Comment:
Not sure if this matters, the changelog says `v8.10.x`, but here it's `~8.10.0` (so I don't think it would go any higher than the .0 if I'm reading [this doc](https://docs.npmjs.com/about-semantic-versioning) correctly)